app/models/generic_event/move_lockout.rb

```diff
@@ -10,6 +10,11 @@ class MoveLockout < GenericEvent
     include LocationValidations
     include LocationFeed
 
+    def trigger(dry_run: false)
+      eventable.is_lockout = true
+      Allocations::RemoveFromNomis.call(eventable) unless dry_run
```

Comment:
We shouldn't remove the move from an allocation in the case of a lockout, I think this may have been accidentally copied from the move cancel event!

Reply:
Ohh yes! good spot!